src/constants.lua

```diff
@@ -53,9 +53,9 @@ constants.ARNS_NAME_DOES_NOT_EXIST_MESSAGE = "Name not found in the ArNS Registr
 constants.UNDERNAME_LEASE_FEE_PERCENTAGE = 0.001
 constants.UNDERNAME_PERMABUY_FEE_PERCENTAGE = 0.005
 constants.PRIMARY_NAME_REQUEST_COST = constants.ARIOToMARIO(10) -- 10 ARIO
-constants.gracePeriodMs = constants.defaultEpochDurationMs * 14 -- 14 epochs
+constants.gracePeriodMs = constants.twoWeeksMs
```

Comment:
Nit that the previous representation dealt with epic counts directly whereas this value coincidentally works out to the correct number of epochs.

Reply:
grace period has no affiliation with epochs